gwt-uibinder-processor/src/main/java/org/gwtproject/uibinder/processor/AbstractFieldWriter.java

```diff
@@ -232,8 +236,35 @@ public void writeFieldDefinition(IndentedWriter w, OwnerField ownerField, int ge
         initializer = String.format("(%1$s) %2$s.create(%1$s.class)",
             getQualifiedSourceName(), UiBinderApiPackage.current().getGWTFqn());
       } else {
-        initializer = String.format("new %1$s()",
-            getQualifiedSourceName());
+        if (type != null) {
+          if (type.getKind().equals(ElementKind.INTERFACE) ||
+                  MoreTypes.asElement(type).getModifiers().contains(Modifier.ABSTRACT)) {
+            StringBuffer sb = new StringBuffer();
+
+            TypeElement element = (TypeElement)MoreTypes.asElement(type);
+            sb.append(MoreElements.getPackage(element).getQualifiedName().toString());
```

Comment:
Can be replaced with:
`sb.append(AptUtil.getPackageElement(element).getQualifiedName().toString())`